openassessment/assessment/models/base.py

```diff
@@ -141,7 +141,7 @@ def structure_hash_from_dict(rubric_dict):
             for criterion in rubric_dict.get('criteria', [])
         ]
         canonical_form = json.dumps(structure, sort_keys=True)
-        return sha1(canonical_form).hexdigest()
+        return sha1(canonical_form.encode('utf-8')).hexdigest()
```

Comment:
Although this change not require for bom-415 but i did it to avoid any future issue. Please let me know if not require.

Reply:
same as above.